chart.js/chart.js.d.ts

```diff
@@ -0,0 +1,526 @@
+// Type definitions for Chart.js 2.6
```

Comment:
Don't we use Chart.js 2.7.0 in the master branch currently?

Reply:
Yep! I haven't really changed the typings to work with any new 2.7 features; this is the latest version of the typings available.